src/features/beacons/selectors.ts

```diff
@@ -10,7 +12,7 @@ import { selectOrdered } from '~/utils/collections';
  * on the UI.
  */
 export const getBeaconsInOrder = createSelector(
```

Comment:
Even though this might seem overzealous, I'd still annotate this as `AppSelector<Beacon[]>`, which I would much rather encounter later on instead of the one inferred by TypeScript:
```ts
const getBeaconsInOrder: ((state: EmptyObject & {
    alert: AlertSliceState;
    beacons: BeaconsSliceState;
    clocks: ClocksSliceState;
    connections: ConnectionsSliceState;
    ... 36 more ...;
    workbench: WorkbenchSliceState;
}) => Beacon[]) & OutputSelectorFields<...> & {
    ...;
}
```